start_ocean.sh

```diff
@@ -10,6 +9,14 @@ export REUSE_DATABASE="false"
 export KEEPER_NETWORK_NAME="ganache"
 export ARTIFACTS_FOLDER=~/.ocean/keeper-contracts/artifacts
 
+# default to `stable` versions of all components
+# unless `--latest` argument is given
+if [ "$1" == "--latest" ] || [ "$2" == "--latest" ]; then
```

Comment:
IMO the easiest way to manage optional arguments is to manage the with `shift`, so you don't need to worry if the argument is passed in any particular order. In this [tutorial](https://mywiki.wooledge.org/BashFAQ/035) there is a good example.

Reply:
was an excellent idea, thanks! I just rewrote the whole script and removed those pesky if/else statements